HIP/hip-1056.md

```diff
@@ -0,0 +1,1769 @@
+Aggregated signatures allow the verification of multiple node signatures to be processed using a single public key.
+Therefore, a user can verify that a block was signed by a sufficient weight within the network by verifying a single
+aggregated signature. TSS-BLS signatures were selected because of their ability to support partial share signatures
+assigned on a weighted number of shares per node with a threshold value required for verification. Additionally,
+adopting TSS-BLS allows the network to utilize a single semi-permanent public ledger id to verify the signature. This
+offers a valuable system optimization as no matter how much the network node membership changes, the same ledger id can
+be used to verify the aggregated signature. TSS-BLS signatures are also reasonably efficient when used within smart
+contracts. A state proof based on this signature can be verified by a smart contract on another ledger for little cost.
+
+A subsequent HIP will be created to outline the changes and justification required for implementing Hedera TSS-BLS
+signatures.
+
+#### Design for State Changes
+
+Including state changes within the block stream is a significant foundational step towards enhancing the
+decentralization of the network. This inclusion allows for the development of alternative node implementations and
+enables downstream state-related services such as state proofs, state snapshots, and reconnect services. It also
+provides a low latency feed of the contents of state as it changes to applications. This can open doors to exciting new
+latency critical application ideas and concepts that could not be built before.
+
+To enable this, state in the consensus node will be restructured into simple named states. This allows for state changes
+to be expressed as simple CRUD operations on named states. This is so downstream consumers do not have to understand the
+full complexity of the merkle state tree.
+
+## Personas and User Stories
+
+**Personas:**
+- _Downstream Developer_: A provider of value-added services utilizing block streams
+- _Verifier_: Any person or service required to verify the integrity of a block or the information contained within a
+  block.
+
+**User Stories:**
+1. As a downstream developer, I want a streamlined feed that contains all event, transactional, and state details of
+   the network so that I can offer value-added services to the ecosystem.
+2. As a downstream developer, I want each block of data to be verifiable with a single, aggregated signature, so that I
+   can authenticate transactional data without the cost and complexity of downloading multiple signature files for
+   verification.
+3. As a verifier for a network, I want a single, self-contained stream of data with aggregated signature signed by a
+   threshold of stake weights, so that I can cost-effectively confirm that the stream represents the consensus output of
```

Comment:
We should really decide what terms we are going to use. Some places use `stake weights` and others simply use `weight`. Either is fine as long as it is used consistently. Inconsistent use will contribute to confusion.